pimd/pim_cmd.c

```diff
@@ -5274,12 +5281,39 @@ static void show_multicast_interfaces(struct pim_instance *pim, struct vty *vty)
 		}
 
 		ifaddr = pim_ifp->primary_address;
+		if (uj) {
+			json_row = json_object_new_object();
+			json_object_string_add(json_row, "name", ifp->name);
+			json_object_string_add(json_row, "state", if_is_up(ifp) ?
+						"up" : "down");
+			json_object_string_add(json_row, "address",
+				inet_ntoa(pim_ifp->primary_address));
+			json_object_int_add(json_row, "ifi", ifp->ifindex);
```

Comment:
ifIndex would be consistent with other parts of the code

Reply:
Done